docs/snippets/common/storybook-test-runner-a11y-config.js.mdx

```diff
@@ -0,0 +1,22 @@
+```js
+// .storybook/test-runner.js
+
+const { injectAxe, checkA11y } = require('axe-playwright');
+
+module.exports = {
+  /* Hook to execute before a story is rendered.
```

Comment:
Rather than these noisy, redundant comments, what about a link (in a comment above `module.exports = {`) to the 'Test hook API (experimental)' section of `test-runner.md`?